frontend/src/hooks/useActions.ts

```diff
@@ -167,8 +167,20 @@ const useActions = (registerHotkeys = false) => {
       handler: () => dispatchCustomEvent('exportImage', { type: 'png', clipboard: true })
     },
     EXPORT_AS_JFLAP: {
-      disabled: () => true,
-      handler: () => console.log('Export JFLAP')
+      hotkeys: [{ key: 's', shift: true, alt: true, meta: true }],
+      handler: () => {
+        // Pull project state
+        const project = useProjectStore.getState().project
+
+        // Create a download link and use it
+        const a = document.createElement('a')
+        // Convert current project to JFLAP format
+        const jflapFile = new Blob([convertAutomatariumToJFLAP(project)])
+        a.href = URL.createObjectURL(jflapFile)
+        // File extension explicitly added to allow for file names with dots
+        a.download = project.meta.name.replace(/[#%&{}\\<>*?/$!'":@+`|=]/g, '') + '.jff'
+        a.click()
+      }
```

Comment:
This just duplicates `SAVE_FILE_AS` handler with the difference being how the blob is made and the file extension no?

Reply:
Yeah it does, ig I could've extracted that logic but wasn't really sure where to put it since a lot of the logic is just kind of inline with the respective actions

Reply:
yeah fair, guess not biggest deal